apps/web/components/tab-navigation.tsx

```
@@ -0,0 +1,52 @@
+"use client";
+
+import React, { useState } from "react";
```

Biome (1.9.4)

[error] The default import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)